styles.css

```diff
@@ -0,0 +1,50 @@
+/* Add a black background color to the top navigation */
+.topnav {
+    background-color: #333;
+    overflow: hidden;
+    viewport-fit: top;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.topnav a:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+/* Add a color to the active/current link */
+.topnav a.active {
+    background-color: #04AA6D;
+    color: white;
+}
+
+.project-tile {
+    text-decoration: center;
+}
+
+header {
+    position: fixed;
+    top: 0px;
+}
+
+
+
+
+user agent stylesheet nav {
+    display: block;
+}
+
+@media only screen and (max-width: 600px) {
```

Comment:
I kindly suggest you not use media query for this case since it is not needed. You can just set the body property without a media query in this case. If there is a layout issue, kindly refer to the comment I made for the fixed unit as the correction.